### PR TITLE
Initial commit of vpnc version 0.5.3

### PR DESCRIPTION
**Pull request by jimklimov**
The IPsec capable VPN client. Or so it claims in multiple OSes ;)
Verified against a Cisco IPsec VPN concentrator (model unknown).

### CODE REVIEW

File: components/network/vpnc/vpnc.p5m [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+file path=usr/sbin/vpnc
+# Scripts
+file path=usr/bin/pcf2vpnc
+file path=usr/sbin/vpnc-disconnect
+# Docs
+file path=usr/share/doc/vpnc/COPYING
+file path=usr/share/man/man1/cisco-decrypt.1
+file path=usr/share/man/man1/pcf2vpnc.1
+file path=usr/share/man/man8/vpnc.8
+file etc/vpnc/default.conf path=etc/vpnc/default.conf.example
```
Comment (xen0l) [inline components/network/vpnc/vpnc.p5m]: Please, ship example file in /usr/share/examples/vpnc

File: components/network/vpnc/vpnc.p5m [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+license $(COMPONENT_LICENSE_FILE) license='$(COMPONENT_LICENSE)'
+
+# This script-package dependency is not as apparent to pkg tools as link-time dependencies ;)
+# Note that unlike the vpnc-script right in this source, that one seems more developed over time
+depend type=require fmri=network/vpnc-scripts
+depend type=require fmri=driver/network/tun
+depend type=require fmri=driver/network/tap
+
+# Bins
+file path=usr/bin/$(MACH64)/cisco-decrypt
```
Comment (xen0l) [inline components/network/vpnc/vpnc.p5m:36]: Do we need 64bit bins here?

**Review by xen0l — CHANGES_REQUESTED** [review]

**Review by xen0l — APPROVED** [review]

File: components/network/vpnc/vpnc.p5m (hunk 2 above)
Comment (jimklimov) [reply components/network/vpnc/vpnc.p5m:36]: per IRC, this is preparation for the 64-bitness of the OS, and the Makefile needed some coercing to do the 64-bit builds/installs into correct paths, so I'd rather keep it (to not redo it later)

**Comment by xen0l** [comment]
This change has been marked for inclusion. Please, squash commits.

**Comment by jimklimov** [comment]
Squashed

**Comment by xen0l** [comment]
Squash commits

**Comment by jimklimov** [comment]
Squeak!.. Whoosh!...

**Pull request merged**